backfiller/backfiller/main.py

```diff
@@ -250,8 +252,15 @@ def get_nodes(self):
 						nodes[substrs[0]] = substrs[1]
 
 		if self.node_database is not None:
-			self.logger.info('Fetching list of nodes from {}'.format(self.node_database))
-			# TODO query the database
+			self.logger.info('Fetching list of nodes from {}'.format(
+				urlparse.urlparse(self.node_database).hostname))
+			conn = self.db_manager.get_conn()
```

Comment:
This is going to make a new conn every time, which isn't ideal. The way most of our other components do this is that main() creates the dbmanager then passes the class a single conn to use.